### PR TITLE
Tidy axe-windows references

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
Fix a few axe-windows references that were missed during the refactoring
- Rename AccessibilityInsights.*.fakes to Axe.Windows.*.fakes (for files that moved)
- Remove references to axe-windows assemblies from the overview
- Remove docs\RulesOverview.md (it belongs in axe-windows)
- Reference axe-windows for rule-related FAQ items

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: docs/Overview.md [hunk 1]
```diff
@@ -13,25 +13,6 @@ Accessibility Insights takes the data provided by UIA and compares it to a set o
...
-#### Runtime components
-These assemblies provide the interaction with UIA, as well as layers that allow the application to interact with that information in a more structured manner:
-
-Assembly | Responsibility
---- | ---
-Axe.Windows.Actions | Provide a high-level set of Actions that are the primary interface into the Runtime components.
-Axe.Windows.Core | Provide data abstractions which represent accessibility data in a platform-agnostic way.
-Axe.Windows.Desktop | Provide platform-specific (Windows) implementations of the platform-agnostic data abstractions. The low-level interactions with UIA occur in this assembly.
-Axe.Windows.Telemetry | Provides an interface which any caller can provide to capture telemetry from Axe.Windows
-AccessibilityInsights.Win32 | Provide a wrapper around Win32-specific code that is needed by other assemblies.
```
Comment (jalkire) [inline docs/Overview.md]: Don't we still have AccessibilityInsights.Win32?

Comment (DaveTryon) [reply docs/Overview.md]: Oops! Re-added in iteration 8

---
In reply to: [290824275](https://github.com/microsoft/accessibility-insights-windows/pull/385#discussion_r290824275) [](ancestors = 290824275)

**Comment by DaveTryon** [comment]
> Project | Responsibility

This section still references AxeWindows-CI

---
Refers to: docs/Overview.md:52 in a394d69. [](commit_id = a394d69561f492c2b771fb9d5c236fa159b45138, deletion_comment = False)

**Comment by DaveTryon** [comment]
> Project | Responsibility

Fixed in iteration 9

---
In reply to: [499228800](https://github.com/microsoft/accessibility-insights-windows/pull/385#issuecomment-499228800) [](ancestors = 499228800)

---
Refers to: docs/Overview.md:52 in a394d69. [](commit_id = a394d69561f492c2b771fb9d5c236fa159b45138, deletion_comment = False)

**Pull request merged**